utils/include/LinuxUtils.cpp

```diff
@@ -81,6 +81,74 @@ namespace PlatformUtils
 		}
 		return false;
 	}
+
+	bool TryGetLogicalDisksNames(std::vector<std::string>& all_disks_names)
+	{
+		constexpr char path_dev[] = { "/dev" };
```

Comment:
I tried to profile unsing constexpr char[] vs const char[] vs std::string, and results were:
- in non optimized mode: string is slow, const and constexpr char[] was equal;
- what's intresting, in optimized mode(O3) string was faster than const and constexpr char[] (which is still equal).
So, i can say that using constexpr is unnecessary, but, also, we need to investigate it(especially std::string results).

Reply:
Would you like to make a research about this theme with me? For now I won't change it